client/src/components/manageProjects/editMeetingTimes.js

```diff
@@ -106,6 +106,13 @@ const EditMeetingTimes = ({
         };
       }
 
+      if ("checkInReady" in values) {
+        theUpdatedEvent = {
+          ...theUpdatedEvent,
+          checkInReady: values.checkInReady,
+        };
+      }
+
```

Comment:
This section updates `handleEventUpdate()` in the same style as the original function, allowing minimal change for us to update `checkInReady` in the db.